ecosystem/resources/members/piQture.toml

```
@@ -0,0 +1,11 @@
+name = "piQture"
+url = "https://github.com/SaashaJoshi/piQture"
+description = "piQture is an open-source Python toolkit designed to simplify the development, execution, and training of Quantum Machine Learning (QML) models tailored for image processing tasks. This library seamlessly integrates with the Qiskit SDK, providing a convenient and user-friendly workflow for leveraging the potential of quantum computing for advanced image processing."
```

This description needs to be < 135 characters. What do you think of this?

```suggestion
description = "Develop, execute, and train Quantum Machine Learning models tailored for image processing tasks. Seamlessly integrated with Qiskit."
```

---

@SaashaJoshi just bumping this in case you missed it

---

This sounds good. 
Sorry, since a bot generated this PR, do I make the changes to the tagged issue or directly change the file in desktop mode?